src/components/Authentication/Authentication.tsx

```diff
@@ -21,9 +21,11 @@ const Authentication: FC = ({ children }) => {
     authenticateUser()
       .then((response) => {
         setUser({ status: 'loaded', isOrgAdmin: response.identity.user.is_org_admin });
+        window.insights?.chrome?.hideGlobalFilter();
```

Comment:
Yeah I was thinking that re: it being a bit strange putting it in authentication; curiosity-frontend has it that way as well. The thing is that the authenticateUser method has to resolve before this is called or it won't work.  

We can put it in each page at the top as well - e.g., in SatelliteManifestPage on line 16 - it just would have to be repeated for each page. Since we just have a few pages at the moment that is an ok route too. LMK if you want me to move it there.

I just updated it to a finally block.

Reply:
I don't think I'd want it on every page even though that may make a bit more sense.  I guess we can leave it for now.